build_tools/embed_data/build_defs.bzl

```diff
@@ -29,17 +29,17 @@ def cc_embed_data(
 
     Generates a header like:
       namespace iree {
-        struct FileToc {
+        struct iree_file_toc_t {
```

Comment:
make this `typedef struct { ... } iree_file_toc_t;`

Reply:
(or ignore for now and can be done as part of the C++->C)

Reply:
Yeah if we can do that later, that would be nice. What's the difference though?

Reply:
Ah this means you don't have to write "struct iree_file_toc_t" every time, yes? Yeah let's save that for later :-)